src/v/config/configuration.cc

```diff
@@ -3664,11 +3664,11 @@ configuration::configuration()
       "iceberg_enabled",
       "Enables the feature that translates topic data to Iceberg tables. When "
       "set to true, this enables the feature at the cluster level and requires "
-      "individual topics to explicitly opt in by setting the iceberg.enabled "
-      "topic-level property to true. When set to false, this disables the "
-      "feature at the cluster level, effectively turning it off for all "
-      "topics.",
-      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      "individual topics to explicitly opt in by setting the "
```

Comment:
Suggestion for this description:

> Enables the translation of topic data into Iceberg tables. Setting iceberg_enabled to true activates the feature at the cluster level, but each topic must also set the redpanda.iceberg.enabled topic-level property to true to use it. If iceberg_enabled is set to false, the feature is disabled for all topics in the cluster, overriding any topic-level settings.

Reply:
cc: @Deflaimun who wrote the original config description, I can add Jake's suggestions but just tagging Paulo so we are all on the same page.

Reply:
+1 for Jake's suggestion. Thanks @bharathv !

Reply:
Done.